docs/kratos/social-signin/35_gitlab.mdx

```diff
@@ -4,8 +4,76 @@ title: Add GitLab as a social sign-in provider in Ory
 sidebar_label: GitLab
 ---
 
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+
 # GitLab
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add GitLab as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the GitLab logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a GitLab OAuth2 Application](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile),
+  noting the following points:
+    - Paste the saved **Redirect URI** from Ory into the **Redirect URI** field in the GitLab form.
+    - Enable the relevant **Scopes** for the application, by ticking the relevant boxes in the GitLab form:
+        - `read_user`
+        - `openid`
+        - `profile`
+        - `email`
+5. After clicking **Save application** on the GitLab form, you see a summary of the registered application's properties.
+    :::caution
+    Do not click away from this page before copying the **Secret** field.
+    This is the only opportunity you will get to copy the secret.
+    :::
+6. Copy the **Application ID** from the registered application in GitLab to the **Client ID** field in the form on the Ory Console.
+7. Copy the **Secret** from the registered application in GitLab to the **Client Secret** field on the Ory Console.
+8. In the **Scopes** field of the form on the Ory Console, add the following scopes (one by one):
+    - `read_user`
+    - `openid`
+    - `profile`
+    - `email`
+9. In the **Data Mapping** field of the form on the Ory Console, add the following Jsonnet code snippet:
+   ```jsonnet
```

Comment:
Similar to the admonitions, code blocks need newlines before and after they're introduced.